File: Introduccion a Poo/Ejercicio 3/Java/Coche.java
public class Coche {
    private String marca;
    private String modelo;
    private int velocidad;
    public Coche(String marca, String modelo, int velocidad) {
        this.marca = marca;
        this.modelo = modelo;
        this.velocidad = velocidad;
    }
    public void acelerar() {
        velocidad = velocidad + 10;
    }
    public void frenar() {
        velocidad = velocidad - 5;
    }

File: Introduccion a Poo/Ejercicio 3/Python/Coche.py
class Coche:
    def __init__(self, marca, modelo, velocidad):
        self.marca = marca
        self.modelo = modelo
        self.velocidad = velocidad
    def acelerar(self):
        self.velocidad = self.velocidad + 10
    def frenar(self):
        self.velocidad = self.velocidad - 5
c1 = Coche("Toyota", "Corolla", 0)
c2 = Coche("Ford", "Focus", 20)
c1.acelerar()
c1.frenar()
c2.acelerar()
c2.frenar()
print(c1.velocidad)
print(c2.velocidad)
